### ISSUE TITLE
Spork decision: Will work with rspec2 and rails 3 on Windows.

### ISSUE THREAD

**Issue by kotp** [marcric/roodle_lms#19]
Please determine if this site provides the answers for Rspec 2 and Rails 3.

Don't forget the win32-process gem for Windows.

The site is here: http://chrismdp.github.com/2010/11/getting-spork-working-now-on-rails-3-rspec-2-and-cucumber/

**Comment by marcric** [comment]
Vic and Folks,

I don't know exactly which of these gems screwed my environment.

But after my attempts on get Cucumber, Rspec and spork working I don't even could run a migration in any other project anymore with similar symptoms.

BTW, did you read this:
http://railstutorial.org/chapters/static-pages#sec:spork

So, I go for the monkey patch which, by the way, is very simple.

Just delete this file in your Ruby folder:

D:\Ruby192\lib\ruby\gems\1.9.1\specifications\rake.gemspec

But that is not the point...

The points are: will we really need this? It is most required when using autotest.

In other words, to solve a problem caused primarily by the way autotest works.

Do we have better autotest alternatives?

http://www.google.com/search?q=autotest+alternatives&ie=utf-8&oe=utf-8&aq=t&rls=org.mozilla:en-US:official&client=firefox-a

**Comment by kotp** [comment]
Kind of answered this in the other testing ticket.

I don't think it is really an issue so much.  It will end up not being one later as we end up with more features anyway.  So I think at this point, I am in agreement that spork doesn't need to be installed.
